cmd/fleetctl/apple_mdm.go

```diff
@@ -202,29 +201,15 @@ func generateMDMAppleBMCommand() *cli.Command {
 			publicKeyPath := c.String("public-key")
 			privateKeyPath := c.String("private-key")
 
-			const cn = "FleetDM"
-
-			// Apple doesn't check the expiry of the public key certificate. It does not matter what value we use.
-			const validityDays = 30
-
-			fmt.Fprintf(
-				c.App.Writer,
-				`Generating Apple Business Manager public key and private key...
-
-`,
-			)
-
-			key, cert, err := tokenpki.SelfSignedRSAKeypair(cn, validityDays)
+			publicKeyPEM, privateKeyPEM, err := apple_mdm.NewDEPKeyPairPEM()
```

Comment:
by using `apple_mdm.NewDEPKeyPairPEM` directly here, it means that we're not enforcing admin-only access here, should we use the API instead?

if it doesn't make sense to enforce here, why it does in the API?

Reply:
Good point. Since this doesn't actually modify anything in fleet, maybe we should remove auth on it.